docs(api): adds docs for function generating the script src (#3033)
webpack/webpack.js.org: src/content/api/module-variables.md

Access to the internal object of all modules.

> ### `__webpack_get_script_filename__` (webpack-specific)

Reviewer: It is possible to specify the type of the subject with arguments, like here https://webpack.js.org/configuration/dev-server#devserverbefore

```suggestion
### `__webpack_get_script_filename__` (webpack-specific)

`function (chunkId)`
```

> This is a function that, when called with a chunk id, returns the filename of that chunk.

Reviewer: And the description will be much cleaner:
```suggestion
Returns the filename of chunk by Id.
```

> T> It is assignable, which allows changing the filename used by the runtime for chunk loading.

Reviewer: Maybe give an example instead? Reading this, i think it is not making it easier for the readers

Reviewer 2: Yes, please

Access to the internal object of all modules.

### `__webpack_get_script_filename__` (webpack-specific)

This is a function that, when called with a chunk id, returns the filename of that chunk.

> 

Reviewer 3: ```suggestion
```

Author: This is newline is necessary for `T>` to work.

> T> `__webpack_get_script_filename__` is assignable, which allows changing the filename used by the runtime for  chunk loading.

Reviewer 2: Please remove space in `for  chunk`

> T> `__webpack_get_script_filename__` is assignable, which allows changing the filename used by the runtime for  chunk loading.

Reviewer 3: ```suggestion
It is assignable, which allows changing the filename used by the runtime for chunk loading.
```

Author: Thank you
